CMakeLists.txt

```
@@ -198,6 +198,13 @@ ENABLE_TESTING()
 ADD_TEST(NAME unit_tests
         WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/src/test"
         COMMAND "${CMAKE_CURRENT_BINARY_DIR}/bin/unit_tests")
+ADD_TEST(NAME trailing_space
+        WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/src/test"
+        COMMAND "python3" "${CMAKE_CURRENT_SOURCE_DIR}/src/test/check_for_no_trailing_space_in_output.py"
+        "--exe" "${CMAKE_CURRENT_BINARY_DIR}/bin/solvitaire"
+        "--gametype" "klondike"
+        "--board" "${CMAKE_CURRENT_SOURCE_DIR}/resources/deal-presets/klondikeSM1.json"
```

Could we possibly change this to 'deal' rather than 'board', just to be consistent across the project?
```suggestion
        "--deal" "${CMAKE_CURRENT_SOURCE_DIR}/resources/deal-presets/klondikeSM1.json"
```